lib/dry/validation/messages/resolver.rb

```diff
@@ -51,18 +51,6 @@ def call(message:, tokens:, path:, meta: EMPTY_HASH)
         end
         alias_method :[], :call
 
-        # Resolve a message
-        #
-        # @return String
-        #
-        # @api public
-        def message_text(message, path, locale: nil, full: false, **opts)
```

Comment:
message_text was marked as public on the API documentation:

https://rubydoc.info/gems/dry-validation/1.5.3/Dry/Validation/Messages/Resolver#message_text-instance_method

For compatibility reason a changing the scope to private and their signature would potentially break systems using this internally.

Reply:
That's a good point. I'll return the method back with the original signature, so the bug fix could be shipped in a patch version.

There is something to consider though:
- the interface is not consistent
`#message` and `#call` expect `path` as a keyword argument
`#message_text` expects it as a positional argument

- should `#message` and `#message_text` be public at all? Isn't it enough to have one public `#call` method?

Reply:
@stind these shouldn't be public, it's a mistake in the yard docs, so don't worry about it. I'm 100% sure nobody is using this.